images/java/Dockerfile-java-alpine

```diff
@@ -0,0 +1,52 @@
+# ---- Build Stage ----
+FROM maven:3.9.4-eclipse-temurin-21-alpine AS build
+
+# Set the current working directory inside the image
+WORKDIR /app
+
+# Copy maven executable to the image
+COPY mvnw .
+COPY .mvn .mvn
+
+# Copy the pom.xml file
+COPY pom.xml .
+
+# Build all the dependencies in preparation to go offline.
+# This is a separate step so the dependencies will be cached unless
+# the pom.xml file has changed.
+RUN mvn dependency:go-offline -B
+
+# Copy the project source
+COPY src src
```

Comment:
Might do "COPY . ." instead. Sometimes there's files in other directories they'll want. Just a thought, anyways.